src/app.service.ts

```diff
@@ -46,105 +68,120 @@ export class AppService {
       .post(
```

Comment:
With Nest, to make your code more robust, make API calls with a try-catch function